experiments/standalone/Bucket/global_bucket.jl

```diff
@@ -0,0 +1,200 @@
+import SciMLBase
+using CairoMakie
+using Dates
+
+import ClimaTimeSteppers as CTS
+using ClimaCore
+using ClimaCore: Remapping
+import CLIMAParameters as CP
+import ClimaComms
+import ClimaLSM
+include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"));
+using ClimaLSM.Bucket: BucketModel, BucketModelParameters, BulkAlbedoFunction
+using ClimaLSM.Domains: coordinates, Column
+using ClimaLSM:
+    initialize,
+    make_update_aux,
+    make_exp_tendency,
+    make_set_initial_cache,
+    PrescribedAtmosphere,
+    PrescribedRadiativeFluxes
+using ClimaLSM.TimeVaryingInputs
+
+function compute_extrema(v)
+    maxes = [maximum(u) for u in v]
+    mins = [minimum(u) for u in v]
+    return (minimum(mins), maximum(maxes))
+end
+
+FT = Float64;
+context = ClimaComms.context()
+earth_param_set = create_lsm_parameters(FT);
+outdir = joinpath(pkgdir(ClimaLSM), "experiments/standalone/Bucket")
+α_sfc = (coordinate_point) -> 0.2;
+α_snow = FT(0.8);
+albedo = BulkAlbedoFunction(α_snow, α_sfc);
+σS_c = FT(0.2);
+W_f = FT(0.15);
+z_0m = FT(1e-2);
+z_0b = FT(1e-3);
+κ_soil = FT(0.7);
+ρc_soil = FT(2e6);
+τc = FT(3600);
+t0 = 0.0;
+tf = 7 * 86400;
+Δt = 3600.0;
+
+bucket_parameters = BucketModelParameters(
+    κ_soil,
+    ρc_soil,
+    albedo,
+    σS_c,
+    W_f,
+    z_0m,
+    z_0b,
+    τc,
+    earth_param_set,
+);
+
+soil_depth = FT(3.5);
+bucket_domain = ClimaLSM.Domains.SphericalShell(;
+    radius = FT(6.3781e6),
+    depth = soil_depth,
+    nelements = (10, 10),
+    npolynomial = 1,
+    dz_tuple = FT.((1.0, 0.05)),
+);
+ref_time = DateTime(2005);
+
+# Precipitation:
+precip = (t) -> 0;
+snow_precip = (t) -> -5e-7 * (t < 1 * 86400);
+# Diurnal temperature variations:
+T_atmos = (t) -> 275.0 + 5.0 * sin(2.0 * π * t / 86400 + 7200);
+# Constant otherwise:
+u_atmos = (t) -> 3.0;
+q_atmos = (t) -> 0.001;
+h_atmos = FT(2);
+P_atmos = (t) -> 101325;
+
+bucket_atmos = PrescribedAtmosphere(
+    TimeVaryingInput(precip),
+    TimeVaryingInput(snow_precip),
+    TimeVaryingInput(T_atmos),
+    TimeVaryingInput(u_atmos),
+    TimeVaryingInput(q_atmos),
+    TimeVaryingInput(P_atmos),
+    ref_time,
+    h_atmos,
+);
+
+# Prescribed radiation -- a prescribed downwelling SW diurnal cycle, with a
+# peak at local noon, and a prescribed downwelling LW radiative
+# flux, assuming the air temperature is on average 275 degrees
+# K with a diurnal amplitude of 5 degrees K:
+SW_d = (t) -> @. max(1361 * sin(2π * t / 86400 + 7200));
```

Comment:
Why do you need `@.` here? Is `SW_d` going to be evaluated on an array of `t`s?

Reply:
no need for it, thanks!